plugin.rb

```diff
@@ -182,6 +183,28 @@
     end
   end
 
+  if Search.respond_to?(:on_preload)
+    Search.on_preload do |search, search_query|
```

Comment:
Mirroring the comment from the other PR, are `search` and `search_query` the best names for those variables?

Reply:
Makes sense, updated variable names.